.env.development

```diff
@@ -1,3 +1,5 @@
+VITE_HOST="http://localhost:8080/"
```

Comment:
This is the backend. The current host in dev mode is the vite host. Please change.

Reply:
this is for rewriting the url to get accepted by the API for short URL generation, in dev, this shall be localhost:8080

Reply:
cf V3 API filtering on allowed_hosts:
https://github.com/Geoportail-Luxembourg/geoportailv3/blob/307ca25802495ac96010b76e53bb305513888929/geoportal/vars.yaml#L396